pkg/grpcfactory/server_test.go

```diff
@@ -0,0 +1,133 @@
+package grpcfactory_test
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/scality/cosi-driver/pkg/grpcfactory"
+	cosi "sigs.k8s.io/container-object-storage-interface-spec"
+)
+
+type mockIdentityServer struct {
+	cosi.UnimplementedIdentityServer
+}
+
+type mockProvisionerServer struct {
+	cosi.UnimplementedProvisionerServer
+}
+
+// generateUniqueAddress returns a unique Unix socket address for each test
+func generateUniqueAddress() string {
+	return fmt.Sprintf("unix:///tmp/test-%d.sock", time.Now().UnixNano())
+}
+
+var _ = Describe("gRPC Factory Server", func() {
+	var (
+		ctx               context.Context
+		cancel            context.CancelFunc
```

Comment:
I don't think you need those, if you add the argument `ctx SpecContext` to each test case, it would be canceled automatically when the test case completes, I believe.

Reply:
I keep forgetting that, will do.